api/dimensions.go

```diff
@@ -141,6 +145,21 @@ func getPositiveIntQueryParameter(queryVars url.Values, varKey string, defaultVa
 	return val, nil
 }
 
+// getStringListFromCSVQueryParameter obtains a list of strings from the provided query parameter,
+// by parsing the CSV value. Up to maxNumItems values are allowed.
+func getStringListFromCSVQueryParameter(queryVars url.Values, varKey string, maxNumItems int) (items []string, err error) {
```

Comment:
Tiny note, i would prefer a shorter name for this function. I dont think 'string list' is necessary context, because you can see that from the return types, and I feel CSV is misleading at this level (it makes me think they've passed a .csv file into the query param).

Maybe something like getQueryParamListValues or somethign? still communicating that we're dealing with one varKey that has multiple values, but without being so specific about the types?